Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -53,30 +53,29 @@ public ItSystemUsage()
             AssociatedDataProcessingRegistrations = new List<DataProcessingRegistration>();
         }
 
-        public bool IsActive
+        public bool IsActiveAccordingToDateFields
         {
             get
             {
-                if (!this.Active)
-                {
-                    var today = DateTime.UtcNow;
-                    var startDate = this.Concluded ?? today;
-                    var endDate = DateTime.MaxValue;
+                if (Concluded == null && ExpirationDate == null) 
+                    return true;
 
-                    if (ExpirationDate.HasValue && ExpirationDate.Value != DateTime.MaxValue)
-                    {
-                        endDate = ExpirationDate.Value.Date.AddDays(1).AddTicks(-1);
-                    }
+                var today = DateTime.UtcNow;
+                var startDate = this.Concluded ?? today;
+                var endDate = DateTime.MaxValue;
 
-                    // indgået-dato <= dags dato <= udløbs-dato
-                    return today >= startDate.Date && today <= endDate;
+                if (ExpirationDate.HasValue && ExpirationDate.Value != DateTime.MaxValue)
+                {
+                    endDate = ExpirationDate.Value.Date.AddDays(1).AddTicks(-1);
                 }
-                return this.Active;
+
+                // indgået-dato <= dags dato <= udløbs-dato
+                return today >= startDate.Date && today <= endDate;
             }
         }
 
         /// <summary>
-        ///     Gets or sets Active. (Enforces Active state. For more info: <see cref="IsActive"/>)
+        ///     Gets or sets Active. (Enforces Active state. For more info: <see cref="IsActiveAccordingToDateFields"/>)
```

Comment:
This should point to CheckSystemValidity... has nothing to do with what you are referencing